buildpack/build.go

```diff
@@ -3,7 +3,7 @@ package buildpack
 import (
 	"errors"
 	"fmt"
-	"io"
+	goio "io"
```

Comment:
I don't love aliasing the very family stdlib `io` because it makes my assumptions as I read the code less accurate. On a quick read through I think I know what `io.Copy` does but in this case I would be wrong! Similarly I might unnecessarily ask myself "huh, what is special about `goio.Writer`?"

It seem like this is a somewhat common stylistic preference. From digital ocean [blog](https://www.digitalocean.com/community/tutorials/importing-packages-in-go)

> When renaming imports to avoid a name collision, you should aim to rename the most local or project specific import. For instance, if you had a local package called strings, and you also needed to import the system package called strings, you would favor renaming your local package over the system package. Whenever possible, it’s best to avoid name collision altogether.

I might quibble with that take a little. I would generally preference system -> local -> non-system dependency.

I would be more comfortable with one of
1. Rename our package. It appears to deal with the filesystem specifically so maybe `fsutil` or `fsio` or something?
2. alias our own package